docs/pages/database-access/reference/cli.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Database Access CLI Reference
+description: CLI reference for Teleport Database Access
+---
+
+# Database Access CLI Reference
+
+## tctl auth sign
+
+When invoked with a `--format=db` flag, produces a certificate authority, a
```

Comment:
nit: produces a certificate authority reads as it produces an authority itself and not it's certificate